src/rasta/c/rasta_new.c

```diff
@@ -1782,8 +1636,78 @@ void sr_cleanup(struct rasta_handle *h) {
 
     sleep(1);
     logger_destroy(&h->logger);
+}
+
+#define IO_INTERVAL 10000
```

Comment:
This seems like it is an important parameter -- could you make it more prominent in the code or add a comment?